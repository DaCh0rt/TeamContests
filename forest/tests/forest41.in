789 868
3271 8110
-8649 8859
-324 8977
2235 243
55 -6544
-204 -3876
1900 2671
6092 -8344
-9859 -1132
7853 -4273
2389 -624
-956 -1437
6023 -4049
-3378 -3690
3765 8700
-5542 5475
3630 -6322
-8374 3655
2040 -5163
-7512 -797
2505 -514
-5059 -6126
6709 892
5970 9916
5871 -4555
972 -8044
-3576 6136
-5110 -1503
-8348 -7681
5243 -1390
-6987 -1666
2511 1666
-6329 9747
2911 -8671
-3314 -3614
-1135 -5366
-5154 -2876
7765 -2716
1578 5673
-9879 -4074
-3783 4830
4952 -9841
3846 -1738
-2691 1511
7099 -2703
420 1799
5943 8728
-5943 -1010
-5644 4052
-5438 -4782
9219 -6871
-2153 5347
4348 7810
-6126 7994
-5628 1792
8727 -6705
-5177 -6998
-3214 9351
6149 -8522
-6343 2742
-5403 -6755
4791 -1237
9347 -4359
-9304 4748
-6085 -7
5414 -2419
1550 1687
7481 -445
6437 -9462
7914 4793
-3930 -6127
7504 -4747
-4031 7061
884 -391
-859 -6025
-5076 -7894
7566 -9593
-6229 -3923
2974 3555
-6311 7783
6058 4109
-4787 -5938
6334 -539
-1878 6473
-8667 6355
-2363 -1203
5388 8883
-4740 -5868
-5962 -5601
7830 118
-1784 -5140
1961 1438
-327 7668
4911 -9860
1290 3855
-7599 3495
-8894 -5827
4518 8900
-429 -1760
1174 -7959
-7746 -8068
-3717 -9105
-5266 6985
-8163 2733
-8580 -9548
-4505 -381
-482 2495
197 7105
7124 -1640
3336 -1993
-6097 5521
-8030 -3925
5604 -2871
-6580 -854
-812 -3575
-4780 -1866
6233 8943
8075 -7949
-8886 1140
1452 -8043
6868 -1966
8468 4589
7384 5859
-1632 -3202
-3575 2699
-5321 -729
-2529 -4222
2479 -5461
-4321 -9677
-9027 -4224
1222 4755
-8780 -4811
-2791 -2868
-5381 -3553
-6440 -9056
-5002 9587
-659 4876
-6541 -5606
-2544 -3372
879 -1826
7499 2020
-5988 -5152
-9007 6848
2772 -6098
-8192 2654
-8150 2910
4537 -2983
-3228 -5538
5273 5133
9949 1570
2491 9183
8564 5514
-2701 -6324
509 -3603
-3510 5150
-438 -2577
-3619 -6512
2302 -2895
-7596 2550
-5708 -4425
1737 -8330
-5113 1412
-2445 2541
137 -5888
-9862 -6175
8328 9933
2734 -8142
-5372 -2336
9740 -7245
8206 -4193
-9635 -3434
-1832 2352
-4633 -9577
8370 -6092
2161 -9263
-8379 1575
4238 -6318
-1067 145
-8332 2657
-3049 4607
5991 2529
6009 -5448
-2763 -7945
9667 -7602
489 9176
5922 679
60 4948
-2059 -8943
6410 4235
7618 3274
2525 3981
-215 1945
-3852 9513
-7025 -8406
5464 7678
3422 594
-4776 -4875
-3976 1458
-5192 4725
-8342 1154
8530 -3087
7373 3721
1963 -2170
4040 -1145
1401 -8424
1483 -9149
-6622 5143
6949 -9958
1933 -6593
9429 -4754
-3893 9286
9340 644
1650 -9517
6306 6421
83 -161
3991 6082
-8185 -9961
7915 -1369
4164 -3458
-3693 -7932
-2188 868
-3259 -2028
-606 -1214
2270 -1154
9952 8077
4252 7157
-5807 4311
5227 6859
-1389 6713
-4970 -4937
-3157 3162
-5822 5138
-1349 -1753
2922 61
9605 -3037
8809 2524
-8172 -7449
-4779 -7844
-8031 7546
-6638 7757
-3628 2514
-1926 -5779
-5063 9501
7522 8322
4824 -7447
-4057 -8614
-4173 -4819
4706 6845
-1737 3706
-6715 -241
-2 5483
1354 7629
4476 7786
-9 969
2289 -550
-8935 1598
328 -1757
3738 -2548
-1067 -3122
2607 2421
-6975 4378
7343 -4824
7325 -5027
-6665 7102
6951 7272
5611 -3608
-280 -7898
194 8142
7947 -457
-9443 -8097
8362 8284
2314 2154
444 9746
3103 3727
141 -8346
6442 409
-9933 6044
-3847 -7280
-6916 519
4167 7961
1530 -4877
5732 -4181
5130 5598
-4671 -8940
-7768 -972
9476 904
-2113 -1635
-6288 -4575
-7716 -9223
-3437 5395
-4803 4279
8235 445
-4321 -884
4478 -3694
-1609 5054
-3142 -1409
-735 -8090
-6618 4680
8282 -1413
9005 -4288
-7205 -2141
-512 -9568
-975 6363
9831 2565
1550 4544
-3987 3435
3184 -4405
-996 -940
-120 2977
5254 -5912
895 -1737
5230 -4160
-6370 -6353
1921 -9053
4186 -9600
4124 9441
2692 3380
8168 -738
-2472 1963
4670 8262
-1594 4268
-568 -6834
-7745 7078
1671 3979
9323 -3503
7614 8125
-3558 -8251
8124 6373
1189 925
5990 -6302
877 -1928
-2887 -8600
-8186 -5922
2490 -3934
-9343 2235
-7687 9117
-3044 111
-2777 9519
6200 -2275
7712 -4825
-6120 -2954
1319 7520
-8478 8293
-3439 -1273
-8061 819
-1150 296
-7069 4563
7840 -808
-1233 7953
-6717 4960
9572 -4133
2429 9377
5581 -2070
6338 3162
-3004 9961
9226 3587
7018 -2201
-7316 6037
3260 -444
6039 777
-784 -8705
-5240 -5714
-6602 4311
-1450 9801
-9559 -1885
2895 6840
1887 -6477
2794 -3988
1484 -4125
-4459 -1837
2025 -1287
-5758 -4355
7860 -797
-2530 -3919
8689 -2625
-1226 2417
75 9746
-7454 -1811
-6679 5931
-2220 -833
-6883 3413
-6118 -8736
7138 8165
-660 7971
3252 8534
-8202 6106
1088 -2781
-3395 -8630
6146 -569
8781 -6996
-6490 5133
5867 7344
7375 -202
-7965 3879
9554 -1948
-7819 -3830
2308 -3700
-2187 672
-7751 -2445
-3350 5270
7008 9237
-7491 -6783
6119 2277
3054 -72
-5265 -4081
-3672 -8224
6529 -4324
6712 7104
5942 3192
-8439 -4014
-6488 5776
-1644 -2034
4584 -1875
-7969 -3622
7383 2654
-675 -1385
5082 2145
7550 2301
-8713 -381
8812 -4117
-8520 124
2722 9531
-6568 -4092
-3882 -6846
-8691 -7053
8504 -2539
-197 -2514
-3479 -3588
543 -9163
-245 -5012
5824 -7910
9906 -4652
-2323 1562
-6320 7765
822 9535
4015 -361
9373 -4409
-2205 -9740
5424 -1939
6894 5713
-3933 -8246
6326 9993
-7898 -3250
3214 3209
-9294 -2977
-2977 5901
1795 5631
9748 7040
-9483 3979
-4813 -149
8119 -5666
966 9502
-1223 -4557
3126 7097
8045 7177
3248 -3267
-3816 -2035
-7863 3020
-2025 -7719
6802 -1903
-2480 -762
-1988 -7049
4070 -7612
3544 5216
4402 -2868
4302 9245
2772 4963
-6176 7432
9476 -2765
423 -4219
9536 -8296
434 9883
-2939 -869
8331 -84
-1793 4502
-4573 -7225
2791 1375
7065 2708
1822 8884
-1160 3124
4924 1392
-204 -2805
468 7439
9277 8426
-6176 -2778
8296 6114
1669 -420
2448 3110
-1334 -9379
4455 4019
1469 7197
1159 -4062
3974 5516
2633 416
5208 -141
-1254 -1096
-8585 -4139
8759 1422
-9939 -7614
-2146 -9199
8362 -9997
-6767 9415
-5397 -3988
6557 5715
-2559 2493
2019 2478
-1135 -6170
-6922 -8118
1504 -4923
4569 7393
4971 5765
9780 1817
3948 -1566
-3300 42
7180 -5471
6222 -2025
-2549 1857
1591 -9506
2823 8640
-9453 -8403
1639 7374
-5620 7913
-2199 -7574
4663 -7024
8356 -3978
9835 -6810
-5880 -2066
-2950 -474
-8319 6092
-4980 5
-3536 3307
-2779 -6054
-4318 6662
1618 103
3787 -2633
-3429 -6474
-6696 1580
-6153 -2575
-3329 -4127
-4358 -1307
7880 -7946
3018 9407
-6893 -6098
-7623 -7975
3679 3515
-2839 -6563
-9681 -1070
5593 -9559
2770 -7839
-8330 3465
-2148 -5911
-4706 2667
-535 -4820
-4140 -1596
6598 -380
-1033 -5287
-8139 5754
-4653 -1166
9085 5845
-9929 4153
176 -3493
-4563 -5514
8647 2948
2661 -6027
-7363 -4811
-7127 -3048
-4129 2233
-5181 -5007
4213 4413
6118 8444
9367 3393
1139 7318
-3734 -1681
8265 1158
5483 -5200
-3320 1066
3039 4570
-5856 -8144
-7051 7229
-9704 -5078
476 6395
4577 8040
7076 -965
-4035 6232
2758 -5837
5258 -1621
-2 -9781
-7128 9471
-9963 -1540
8020 6124
-597 6186
-8378 7916
8127 -91
9314 -7447
3763 8496
-6467 -6668
4812 -8750
4434 -6188
4505 -5718
8786 378
8439 -4508
7996 7914
2066 390
-5051 -9967
7094 -755
8186 8404
-3104 587
661 1434
-7965 2419
2220 -1534
9010 694
-1751 -2384
9507 6430
-4668 7668
9493 7759
1181 9156
1710 4549
9678 9537
5497 1872
9722 -7523
5763 7852
-7374 7550
2662 4163
1927 -9716
-816 -8767
-7316 -4864
-7887 9529
6926 -6976
5225 -3044
518 -4107
9536 8440
8173 1786
-6985 6677
-4693 -5140
-4669 -5034
-9478 -948
-9517 -656
5751 1520
-4554 4133
4507 -2804
-4545 5015
-9987 -9883
1683 -7106
-6354 6501
5201 2660
9313 476
4013 -3612
7746 -5886
-5838 -3576
4168 -4902
7139 -8254
-8834 9373
-6283 -1911
5433 -5453
1044 6626
-8025 -9731
-4225 -1451
9407 -5487
6168 1761
4718 -2235
-3543 4131
-1388 8276
-1393 6540
2919 2117
-7151 -7021
3049 4812
-6875 1518
1593 6441
-6756 -9467
5386 -9689
3146 -2752
-3279 2324
177 -846
3883 -2432
9675 -1625
4574 -4040
9845 -3998
7644 -4767
-4725 6076
7042 5370
6331 -59
-8992 3025
-1608 8403
-1526 287
-9317 9999
8200 -2947
-4328 -8412
3355 -2703
-3647 -9359
4947 -5894
-158 -9721
-7602 890
-7566 7473
-6601 3540
-6895 -3113
8802 -2869
6998 8074
6787 1554
-6768 -7866
-8464 -6354
6876 4351
-5894 3603
8164 -7218
-2642 752
-6194 3043
9413 -7579
4754 132
-1112 -1588
-3685 -8551
2997 2595
9008 3921
-360 -6607
-975 -6836
-6864 7801
-5031 7154
-8994 755
3785 4199
3966 -6093
-7319 -6124
7783 -9398
1242 3830
9164 1248
3204 -6551
4124 6382
694 2591
78 -4750
-4100 7663
-4849 1396
7696 -5358
-1164 -5530
-2928 617
9075 -7172
9810 -5849
-3000 4289
1505 -4024
-9786 1369
-3480 4003
3756 -5633
-1377 -8793
7740 -4033
-772 482
532 9250
-7273 -6715
-6378 4113
7864 240
7385 8757
8981 -1495
7056 -2107
4506 -9286
-2359 7748
1428 -2548
-1125 1908
-5370 9035
-5568 3284
2661 -6479
-1775 3309
-5611 4486
3419 9324
8706 -1393
2628 -5801
-5290 -3863
-9956 7517
-1955 3884
469 -8796
4159 -7833
-4506 -3532
5362 -4131
-3056 5070
1363 -2795
477 -6016
-2018 -1914
6056 9828
-8071 2568
-2952 -3659
3715 -9851
-4289 875
9325 -9779
7128 -5699
9074 -7654
-895 -5656
1976 -910
-4730 -72
-2582 -4990
-9154 -2274
1580 9820
9159 -425
-6373 5752
-7443 1430
1007 -8713
