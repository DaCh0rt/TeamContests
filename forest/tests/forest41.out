83194.01 464500919.16
